content/700-optimize/300-recordings.mdx

```diff
@@ -0,0 +1,47 @@
+---
+title: 'Recordings'
+metaTitle: 'Optimize: Recordings'
+metaDescription: "Learn about using Optimize's recording feature."
+tocDepth: 3
+toc: true
+---
+
+The recordings feature helps developers debug and isolate sets of queries into distinct sessions, known as recordings. This targeted approach enables precise performance analysis and optimization by preventing the mixing of queries from different applications or test rounds, leading to clearer insights and more effective debugging.
+
+## Managing a recording session
+
+You can manually start and stop recording sessions via the [Optimize dashboard](https://optimize.prisma.io) by clicking the **Start Recording** and **Stop Recording** buttons.
+
+:::warning
+A recording will automatically stop if the 10k query limit is reached or if the Prisma schema of the app is changed.
+:::
+
+## Identifying a recording session
+
+You can rename and tag your recording sessions for easy identification and context. Click on the default name of the session and type the desired name.
+
+## Data captured in a recording session
+
+Each recording session captures detailed insights about the queries executed in your app, including:
+
+- All queries executed during the session.
+- The raw query generated and sent to the database by Prisma ORM as **Raw Query**.
+- The number of times a query pattern was executed, listed as **Count**.
+- [Query performance metrics](/optimize/performance-metrics).
+- Errors encountered during query execution.
+
+Each recording can include up to 10k queries. There are no limits on storage retention.
+
+## Recommendations from a recording session
+
+When a recording session ends, Optimize generates recommendations such as:
+
+- [Excessive number of rows returned](/optimize/recommendations/excessive-number-of-rows-returned)
+- [Full table scans caused by LIKE operations](/optimize/recommendations/full-table-scans-caused-by-like-operations)
+- [Queries on unindexed columns](/optimize/recommendations/queries-on-unindexed-columns)
+
+:::info
+Use the [AI Explainer](/optimize/ai-explainer) to ask follow-up questions about a recommendation.
```

Comment:
same thing here as above, i wont leave this comment everywhere but once you have an answer please update it accordingly